src/command/upload.rs

```diff
@@ -372,6 +372,15 @@ fn hippofacts_file_path(hippofacts_arg: &str) -> anyhow::Result<PathBuf> {
     }
 }
 
+// The list of filenames to look for has to take case sensitivity
+// into account.
+#[cfg(target_os = "windows")]
+const SPEC_FILENAMES: &[&str] = &["HIPPOFACTS", "hippofacts.toml"];
+// TODO: apparently there is a config option to make Mac filesystems
+// case sensitive; fml.
+#[cfg(target_os = "macos")]
```

Comment:
Curious... Can this be compressed down to

```
cfg(any(target_os = "macos", target_os = "windows"))
```

Or is that not possible?

Reply:
I think it can but I wanted to put the TODO comment on the Mac bit